### ISSUE TITLE
pidstat parser give wrong results without `-H`

### ISSUE THREAD

**Issue by braindevices** [kellyjonbrazil/jc#281]
The problem caused by `Time` field which is not in seconds. This maybe a documentation problem. Apparently jc expecting `-H`:
>  -H     Display timestamp in seconds since the epoch.

In document it does not say anything about `-H`:
> jc - JSON Convert `pidstat -h` command output parser
> 
> Must use the `-h` option in `pidstat`. All other `pidstat` options are
> supported in combination with `-h`.
> 
> Usage (cli):
> 
>    $ pidstat -h | jc --pidstat
> 
> or
> 
>    $ jc pidstat -h

the pidstat output is:
```
Linux 5.4.0-125-generic (test) 	09/03/2022 	_x86_64_	(12 CPU)

# Time        UID       PID    %usr %system  %guest   %wait    %CPU   CPU  minflt/s  majflt/s     VSZ     RSS   %MEM  Command
01:20:32 PM  1000      9182    0.02    0.01    0.00    0.00    0.03     3      1.71      0.00  461128  110744   0.17  kate
```
The json output with `jc --pidstat --pretty` is:
```json
[
  {
    "time": 12032,
    "uid": null,
    "pid": 1000,
    "percent_usr": 9182.0,
    "percent_system": 0.02,
    "percent_guest": 0.01,
    "percent_wait": "0.00",
    "percent_cpu": 0.0,
    "cpu": 0,
    "minflt_s": 3.0,
    "majflt_s": 1.71,
    "vsz": 0,
    "rss": 461128,
    "percent_mem": 110744.0,
    "command": "0.17  kate"
  }
]
```

versions:
```
jc -v
jc version:  1.21.2
python interpreter version:  3.8.10
python path:  /usr/bin/python3

https://github.com/kellyjonbrazil/jc
© 2019-2022 Kelly Brazil

pidstat -V
sysstat version 12.2.0
(C) Sebastien Godard (sysstat <at> orange.fr)

```

**Comment by kellyjonbrazil** [comment]
Thanks for reporting this! I’ll investigate… I wonder if this is a newer version of pidstat than I tested on?

**Comment by kellyjonbrazil** [comment]
Ah yes, it looks like I built this parser with `pidstat` v10.1.5 on CentOS 7. I see v12.4.0 on one of my Ubuntu VMs with the same output as you.

Unless there are any other breaking differences I'll just do a documentation fix for this one. Thanks again!

**Comment by kellyjonbrazil** [comment]
fixed in `dev` and will be released in the next version in a few weeks.

**Comment by kellyjonbrazil** [comment]
merged into v1.22.0
